website/content/ja/events/2024-cloud-native-sustainability-week.md

```diff
@@ -0,0 +1,67 @@
+weight: 94
+slug: cloud-native-sustainability-week
+aliases:
+  - /cloud-native-sustainability-week
+---
+
+<p class="mt-5 mb-5"><img src="/images/cloud-native-sustainability-week-v1-logo.webp" alt="CNCF Cloud Native Sustainability Week Banner"></p>
+
+CNCFサステイナビリティ週間は、[環境持続可能性のためのCNCF Technical Advisory Group (TAG ENV)](http://github.com/cncf/tag-env-sustainability) が [クラウドネイティブコンピューティングファウンデーション (CNCF)](http://cncf.io) と共同で毎年開催するイベントです。このイベントの主な目的は、クラウドネイティブ業界とオープンソース空間における環境持続可能性を促進することです。
+
+このグローバルイベントは、対面のミートアップとバーチャルセッションの両方を含み、世界中のローカルテックコミュニティが主催し、TAG ENVがまとめています。このようなグローバルな大陸間のコラボレーションを奨励することで、テクノロジー分野においてサステイナビリティが不可欠な要素であることを、また、時にはわずかなものであっても、環境を尊重しながら技術的・革新的な取り組みを実施することを、世界中のあらゆる人々が理解できるようになると私たちは信じています。
+
+[サステイナビリティ週間2023](https://tag-env-sustainability.cncf.io/events/cloud-native-sustainability-week-2023/)が成功を収めた後、今年の企画はすでに進行中で、より環境に優しく魅力的な取り組みへの新たな取り組みが約束されています。
+
+**2024年10月の第1週に**、[世界中のCNCFコミュニティグループ](https://community.cncf.io/chapters/)が、クラウドネイティブの持続可能性に焦点を当てたミートアップをそれぞれの都市で開催します。これらのミートアップは、この取り組みを支援したいと考えている方ならどなたでも開催できます。さらに、[CNCF Environmental Sustainability Technical Advisory Group (TAG ENV)](http://github.com/cncf/tag-env-sustainability) がバーチャルミーティングを開催します。バーチャルミーティングと録画された実際のイベントの録画は、YouTubeのプレイリストにアップロードされます。
+
+<div class="embed-responsive embed-responsive-16by9"><iframe class="embed-responsive-item" src="https://www.youtube.com/embed/ezGSYtvQH2c?si=Qz5inM2pI4rHuerm" title="Sustainability Week 2024" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></div>
+
+## ミートアップのロケーション
+
+*このリストは随時更新されます。イベントの数週間前に、完全な概要を公開します！最新の状況や問い合わせ先を確認するには、[こちらのドキュメント](https://docs.google.com/document/d/1xfUqxt0EeMhrsg8h5p_cAJ1ZHzWryrIywue53rVRFJw/edit?usp=sharing)をご覧ください。
+
+<!-- cSpell:disable -->
+| **NR** | **場所** | **日付** | **イベントリンク** | **名前** |
+|---|---|---|---|---|
+| 1 | Amsterdam, Netherlands | TBD | TBD | Brendan Kamp
+| 2 | Guatemala | TBD | TBD | Sergio Méndez
+| 3 | China | TBD | TBD | Sam Yuan
+| 4 | Norway | TBD | TBD | Marta Paciorkowska
+| 5 | Tokyo, Japan | TBD | TBD | Sunyanan Choochotkaew
+| 6 | Milan, Italy | TBD | TBD |  Michel Murabito
+| 7 | Sao Paulo, Brazil | TBD | TBD | Carol Valenica
+| 8 | India | TBD | TBD | Saiyam
+<!-- cSpell:enable -->
+
+## イベントのゴール
+
+1. **現在の[Cloud Native Sustainability Landscape](/landscape/)をより深く理解する**：コミュニティやIT業界全体と関わり、[Cloud Native Sustainability Landscape](/landscape/)をより深く理解します。
+2. **知識のギャップを特定する**：さらなる調査やナレッジ共有が必要なクラウドネイティブの環境持続可能性に関する分野を特定します。
+3. **プロジェクトのコラボレーションを促進**：クラウドネイティブの持続可能性の分野に関連する既存のプロジェクトやツールのさらなる開発と改善に積極的に協力するよう参加者に促す。
+4. **関心を高める**：クラウドネイティブの環境持続可能性に関する関心とナレッジ共有を高める。
+5. **TAG ENVとCNCFを宣伝する**：#tag-environmental-sustainability を宣伝し、CNCFについて広く認知度を高めます。
+6. **調査**：クラウドネイティブの持続可能性に対するエンドユーザーの採用状況に関する情報を収集するためのアンケートを配布する。
+7. **バッジ**：主催者や講演者にcredlyのバッジを提供します。
```

Comment:
```suggestion
7. **バッジ**：主催者や講演者にCredlyのバッジを提供します。
```

IMO: [Credly](https://info.credly.com) should be capitalized since it is a proper noun (the original text would need to be corrected).